src/components/side_panel/conditional_formatting/cf_editor/color_scale_rule_editor.xml

```diff
@@ -37,15 +37,13 @@
         t-if="threshold !== undefined and threshold.type !== 'value'"
       />
       <input type="text" class="o-input o-threshold-value" t-else="" disabled="1"/>
-      <ColorPickerWidget
-        currentColor="getThresholdColor(threshold)"
-        toggleColorPicker="(ev) => this.toggleMenu('colorScale-'+thresholdType+'Color', ev)"
-        showColorPicker="state.openedMenu === 'colorScale-'+thresholdType+'Color'"
-        onColorPicked="(color) => this.setColorScaleColor(thresholdType, color)"
-        title="fill_color"
-        icon="'o-spreadsheet-Icon.FILL_COLOR'"
-        disabled="threshold === undefined"
```

Comment:
If the threshold is None, you can pick a color but nothing happen. It's confusing. Before, the color picker was disabled (but we could also just remove the color picker if it's easier).
![image](https://github.com/odoo/o-spreadsheet/assets/10908305/01e3866e-0998-4a06-8763-8ad3f1439fa2)

Reply:
After discussing with Fracnois, I removed the picker and the input if there was no midpoint in the CF